tests/main/interfaces-accounts-service/task.yaml

```diff
@@ -3,61 +3,23 @@ summary: Ensure that the accounts-service interface works
 # Only test on classic systems with AppArmor DBus mediation
 # Don't test on ubuntu-14.04, since it's gnome-online-accounts daemon
 # seems to be incompatible.
-systems: [ ubuntu-16* ]
-
-environment:
-    XDG_CONFIG_HOME: $(pwd)/config
-    XDG_DATA_HOME: $(pwd)/share
-    XDG_CACHE_HOME: $(pwd)/cache
+systems:
+    - ubuntu-16.04-*
+    - ubuntu-18.04-*
+    - ubuntu-20.04-*
```

Comment:
Could we use `ubuntu-2*` here?

Reply:
Yes but look at what this tells you. The patterns should flip to a specific list as the exclusions will only grow. This applies to several other tests that are similarly in the trap of upstreaming breaking the protocol and the interface really not allowing snaps to work across base os releases

Reply:
I don't follow. If we use `ubuntu-2*` in an ideal world we don't have to touch this at all until 2030?

About the breaking - it seems better to know when this happen. Assuming we use the above pattern and 20.10 comes along and breaks the protocol we will know and can switch to the list as it currently is with a comment that later versions break. if we keep the list as is we will (most likely) forget to add 20.10 when it comes around and never test it and learn about the breakage.

Or am I missing something here?

Reply:
I'm sorry wrong test. I misread the diff (I was looking at various changes I proposed) and there are some that have a pattern like `- -ubuntu18.04`, `- -ubuntu20.04` (because the test only works on 16.04).

My comment is bonkers. Sorry.